westcoast/src/main/java/frc/commands/Commands.java

```diff
@@ -35,6 +35,21 @@ static TeleopDrive raw(@NotNull Definition definition) {
             );
         }
 
+        @NotNull
+        static Command cheesy(@NotNull Definition definition) {
+            Input input = definition.input;
+            return new CheesyDrive(
+                    input.joystick::getX,
+                    input.joystick::getY,
+                    () -> !input.attenuate.get(),
+                    input.turnInPlace,
+                    input.invert,
+                    definition.subsystems.driveTrain,
+                    0.5,
```

Comment:
If these are not accessible by the user, why not have them in the final variable list of the command itself

Reply:
Can be used by different chassis and I think they will be slightly different then. we can move to parameters in the definition in the future. Or they can be part of the function parameters.

Reply:
I'd say put them in the definition as they are a direct function of drivetrain params